src/hash.cc

```diff
@@ -50,10 +53,12 @@ size_t hash_data(const char* input, size_t len)
 
     const uint8_t* tail = data + nblocks * 4;
     uint32_t key = 0;
-    switch (len & 3)
+    switch (len & 0b11)
     {
-        case 3: key ^= tail[2] << 16; /* fallthrough */
-        case 2: key ^= tail[1] << 8;  /* fallthrough */
+        case 3: key ^= tail[2] << 16;
+            [[fallthrough]];
+        case 2: key ^= tail[1] << 8;
+            [[fallthrough]];
```

Comment:
This alignment is inconsistent with how `case 1` aligns statements, which itself is inconsistent with how the code is usually formatted in Kakoune.

Could you make all cases use consistent indentation (either by moving the fallthrough attirbutes back onto the previous line, keeping all operations clearly aligned, or by introducing a line break after all `case N:` and just indenting by 4 spaces as usual.

Reply:
Yeah, my b used to there just being a clang-format I can blindly apply :laughing: